archived_apps/NFT/README.md

```diff
@@ -17,6 +17,10 @@
     </a>
 </p>
 
+> **Warning**
+> This application have been temporarily archived waiting the finalised release of of the [Sway multi-token standard](https://github.com/FuelLabs/rfcs/pull/17) before being updated. In it's current form, it does not comply with this standard and as such should be considered outdated.
+> It is not advised to use this applciation for production purposes, use at your own risk
```

Comment:
```suggestion
> It is inadvisable to use this application for production purposes.
```

All app examples are really at their risk because we do not audit these nor do we aim for security per-say.